go/coordinator/Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: build
 build:
-	go build -v -o bin/chroma ./cmd
+	go build -v -o bin/coordinator ./cmd/coordinator/
```

Comment:
would it make sense to have two independent build commands

Reply:
there are 2 build commands? what do you mean?